### PR TITLE
Elasticache support

### PR DESCRIPTION
**Pull request by noqcks**
I realized that we don't have _any_ support for elasticache. 

A number of our projects already use elasticache, and this will come in handy when I try to make wordpress suck less. 

@telusdigital/guild-devops plz review :100:

### CODE REVIEW

File: defaults/main.yml [hunk 1]
```diff
@@ -16,6 +16,9 @@ dns_load_balanced_roles:
 root_volume_size: 10
 subnet_prefix: '10.0'
 using_rds: no
+using_elasticache: yes
```
Comment (pdufault) [inline defaults/main.yml]: I'm not sure that enabling elasticache by default is a good move -- does this need to be enabled here?

**Comment by pdufault** [comment]
Other than my comment, this looks good.

**Comment by killerwails** [comment]
other than default set to yes, things look good

**Comment by noqcks** [comment]
@killerwails @pdufault It is only enabled when you set `- cache` as one of the `required_systems`, but I agree, it shouldn't be enabled by default for now. The ansible elasticache module is a little flaky right now;I have a few PRs in `ansible-modules-core` that will hopefully fix that, but I agree, for now we shouldn't have it enabled by default.

**Comment by aaronpederson** [comment]
Team approved

**Pull request merged**